string name
float32[2] center
float32 radius
float32 distance
